components/raftstore-v2/tests/integrations/test_transfer_leader.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::time::Duration;
+
+use engine_traits::{OpenOptions, Peekable, TabletFactory};
+use futures::executor::block_on;
+use kvproto::{
+    metapb,
+    raft_cmdpb::{AdminCmdType, CmdType, Request, TransferLeaderRequest},
+};
+use raft::prelude::ConfChangeType;
+use raftstore_v2::router::PeerMsg;
+use tikv_util::store::new_peer;
+
+use crate::cluster::Cluster;
+
+fn put_data(
```

Comment:
do you have test case to verify the lock transfer logic?

Reply:
V1's integration test has tests of it. Once we have v2's integration test, we can reuse them.